hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneHAClusterImpl.java

```diff
@@ -169,6 +169,11 @@ public MiniOzoneCluster build() throws IOException {
         throw new IllegalArgumentException("Number of active OMs cannot be " +
             "more than the total number of OMs");
       }
+
+      // If num of ActiveOMs is not set, set it to numOfOMs.
```

Comment:
Should this be set before the check on line 168?

Also why not set it always. Would that be wrong?

Reply:
If numOfActiveOMs is not set we set with numofOMs (which is total count of OMS). When user set some value to numOfActiveOMs we don't want to override that, this is the reason for doing that.

If it is not set, we don't need to do this check
if (numOfActiveOMs > numOfOMs) {
        throw new IllegalArgumentException("Number of active OMs cannot be " +
            "more than the total number of OMs");
      }
So, it is okay to add after this too.